### PR TITLE
fix(clerk-js,types): Enable the ability to disable preparation on startEmailLinkFlow

### PR DESCRIPTION
**Pull request by tmilewski**
## Description

TODO

Fixes SDK-1799

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: packages/clerk-js/src/ui/components/SignIn/SignInFactorOne.tsx [hunk 1]
```diff
@@ -123,7 +123,6 @@ export function _SignInFactorOne(): JSX.Element {
   }
 
   switch (currentFactor?.strategy) {
-    // @ts-ignore
```
Comment (tmilewski) [inline packages/clerk-js/src/ui/components/SignIn/SignInFactorOne.tsx:126]: Not necessary

**Comment by tmilewski** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @tmilewski - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/backend | 1.2.2-snapshot.v8503eda |
| @clerk/chrome-extension | 1.0.17-snapshot.v8503eda |
| @clerk/clerk-js | 5.6.0-snapshot.v8503eda |
| @clerk/elements | 0.6.0-snapshot.v8503eda |
| @clerk/clerk-expo | 1.2.0-snapshot.v8503eda |
| @clerk/express | 0.0.11-snapshot.v8503eda |
| @clerk/fastify | 1.0.13-snapshot.v8503eda |
| gatsby-plugin-clerk | 5.0.0-beta.45 |
| @clerk/localizations | 2.4.4-snapshot.v8503eda |
| @clerk/nextjs | 5.1.4-snapshot.v8503eda |
| @clerk/clerk-react | 5.2.3-snapshot.v8503eda |
| @clerk/remix | 4.0.14-snapshot.v8503eda |
| @clerk/clerk-sdk-node | 5.0.10-snapshot.v8503eda |
| @clerk/shared | 2.2.2-snapshot.v8503eda |
| @clerk/testing | 1.1.6-snapshot.v8503eda |
| @clerk/themes | 2.1.9-snapshot.v8503eda |
| @clerk/types | 4.6.0-snapshot.v8503eda |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/backend`
```sh
npm i @clerk/backend@1.2.2-snapshot.v8503eda --save-exact
```
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.0.17-snapshot.v8503eda --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.6.0-snapshot.v8503eda --save-exact
```
`@clerk/elements`
```sh
npm i @clerk/elements@0.6.0-snapshot.v8503eda --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@1.2.0-snapshot.v8503eda --save-exact
```
`@clerk/express`
```sh
npm i @clerk/express@0.0.11-snapshot.v8503eda --save-exact
```
`@clerk/fastify`
```sh
npm i @clerk/fastify@1.0.13-snapshot.v8503eda --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.0-beta.45 --save-exact
```
`@clerk/localizations`
```sh
npm i @clerk/localizations@2.4.4-snapshot.v8503eda --save-exact
```
`@clerk/nextjs`
```sh
npm i @clerk/nextjs@5.1.4-snapshot.v8503eda --save-exact
```
`@clerk/clerk-react`
```sh
npm i @clerk/clerk-react@5.2.3-snapshot.v8503eda --save-exact
```
`@clerk/remix`
```sh
npm i @clerk/remix@4.0.14-snapshot.v8503eda --save-exact
```
`@clerk/clerk-sdk-node`
```sh
npm i @clerk/clerk-sdk-node@5.0.10-snapshot.v8503eda --save-exact
```
`@clerk/shared`
```sh
npm i @clerk/shared@2.2.2-snapshot.v8503eda --save-exact
```
`@clerk/testing`
```sh
npm i @clerk/testing@1.1.6-snapshot.v8503eda --save-exact
```
`@clerk/themes`
```sh
npm i @clerk/themes@2.1.9-snapshot.v8503eda --save-exact
```
`@clerk/types`
```sh
npm i @clerk/types@4.6.0-snapshot.v8503eda --save-exact
```

**Comment by tmilewski** [comment]
!snapshot

**Pull request closed without merging**